tutorials/text/usetex.py

```diff
@@ -73,6 +35,34 @@
 ``'monospace'``   ``'Courier'``, ``'Computer Modern Typewriter'``
 ================= ============================================================
 
+The default font is Computer Modern.  All of the other fonts are Adobe fonts.
+Times and Palatino each have their own accompanying math fonts, while the other
+Adobe serif fonts make use of the Computer Modern math fonts.
+
+To enable LaTeX and select a font, use e.g.::
```

Comment:
This isn't terribly clear terminology, so maybe needs a bit more explanation.  Is "font.family": "Helvetica" supposed to be a shortcut or are we claiming "Helvetica" is actually a font family?  To me a "font family" is a set of fonts that provides serif/sans-serif/different weights, etc.  But thats not how we are using it here?  

Given that most folks using usetex are probably familiar with this from latex, is it not helpful to state explicitly what package is used?  

I'm not sure this change is right or desirable, though the previous text could be fixed as well...

Reply:
"Helvetica" is a "font family" (see e.g. https://developer.mozilla.org/en-US/docs/Web/CSS/font-family) (it's a called a family because it can exist in bold/italic/etc.).  Note that this is like setting `font.family: DejaVu Sans` in non-usetex mode.  This terminology is also consistent with tex (see e.g. table 3 in http://mirrors.ctan.org/macros/latex/required/psnfss/psnfss2e.pdf).  I slightly tweaked the wording, does it help?

I made the change because this was describing a relatively more complex way to set the font, whereas a simpler approach is now available.

I agreed that listing the tex packages is probably helpful, now done.